docs/source/top_level_elements/genre.rst

```diff
@@ -9,7 +9,19 @@ This section describes the different values for `genre` that we currently have i
 genre[@authority = 'dct']
 -------------------------
 
-We will explore other options for records using `@authority = 'dct'` or uncontrolled vocabularies, e.g. mapping values as `typeOfResource`. Many of the records in Volunteer Voices have multiple `genre`s; e.g. `volvoices:14311 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:14311/datastream/MODS/view>`_.
+We will explore other options for records using `@authority = 'dct'` or uncontrolled vocabularies, e.g. mapping values as `typeOfResource`. Many of the records in Volunteer Voices have multiple `genre`s; e.g. `volvoices:14311 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:14311/datastream/MODS/view>`_. Following our mapping for :code:`typeOfResource`, we will use the appropriate URI for the given text value.
+
++-------------------------------------+---------------+--------------------------------------------------+----------------+
+| MODS genre[@authority='dct'] String | RDF Predicate | URI                                              | dcterms String |
++-------------------------------------+---------------+--------------------------------------------------+----------------+
+| text                                | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/txt> | Text           |
++-------------------------------------+---------------+--------------------------------------------------+----------------+
+| still image                         | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/img> | StillImage     |
++-------------------------------------+---------------+--------------------------------------------------+----------------+
+| image                               | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/img> | Image          |
++-------------------------------------+---------------+--------------------------------------------------+----------------+
+| collection                          | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/col> | Collection     |
```

Comment:
@CanOfBees - just checking, does the genre element ever have a value of "collection"? If so, it's likely inaccurate. If not, I wonder if we should remove it from the table in this section.

Reply:
@mlhale7 It appears on a compound obj in Vol Voices: 

```
volvoices:12079
volvoices:12078
volvoices:12077
volvoices:12076
```

Reply:
Thanks. The record does seem to attempt to describe all of the three items shared in the compound object, so we can let it go as a collection I guess. This would be problematic for PrimoVE as I'm currently highlighting these items as a "Digital Collection".

Reply:
As we discussed in our meeting, we'll go ahead and make a ticket to remove this since it's atypical.

Reply:
@mlhale7 this has been updated. Thanks!